src/foam/u2/detail/SectionedDetailPropertyView.js

```diff
@@ -262,28 +262,36 @@ foam.CLASS({
 
   requires: [
     'foam.u2.layout.Cols',
-    'foam.u2.layout.Rows'
+    'foam.u2.layout.Rows',
+    'foam.u2.Visibility'
   ],
 
   properties: [
     'prop',
+    {
+      class: 'FObjectProperty',
+      of: 'foam.core.Slot',
+      name: 'visibilitySlot',
+      expression: function(prop) {
+        return prop.createVisibilityFor(this.data$).map((m) => m !== this.Visibility.HIDDEN);
+      }
+    }
   ],
 
   methods: [
     function initE() {
       var self = this;
       this.SUPER();
+
       this
-        .show(this.prop.createVisibilityFor(this.data$)
-          .map(m => m != foam.u2.Visibility.HIDDEN))
+        .show(this.visibilitySlot)
```

Comment:
Probably not a big deal right now but if `prop` changes, this won't react to it. You'll need a proxy slot here if you want that reaction to work. Something like:
`.show(self.ProxySlot.create({delegate$: self.visibilitySlot$})`

Reply:
Resolved by b94d49d.